composer.json

```
@@ -1,12 +1,15 @@
 {
-  "name": "islandora/islandora",
+  "name": "drupal/islandora",
   "description": "Core Islandora module",
   "type": "drupal-module",
   "keywords": ["Drupal", "Islandora"],
   "homepage": "https://github.com/Islandora/islandora",
   "support": {
     "issues": "https://github.com/Islandora/documentation/issues"
   },
+  "replace": {
+    "islandora/islandora": "*"
```

Might make more sense to keep this related to the specific version; otherwise, I think this could end up proposing `drupal/islandora:2` as an update for `islandora/islandora:1` kind of thing (or similarly for other constraints that are not quite compatible, as in, if one was constraining `islandora/islandora` to a specific minor or patch version or the like for whatever reason), as this asserts that `drupal/islandora` is a replacement for _all_ versions of `islandora/islandora`. 

```suggestion
    "islandora/islandora": "self.version"
```